applications/nrf5340_audio/doc/building.rst

```diff
@@ -223,6 +221,36 @@ Complete the following steps to build the application:
    Alternatively, you can add the ``-d`` parameter to the ``west`` command to specify a custom build folder. This lets you build firmware for both
```

Comment:
Should we remove the note on line 191-193 about the script being recommended?

Reply:
Discussed with the team and agreed it is time to remove this note